examples/deploy-ecs-cluster/variables.tf

```
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------
+
+# REQUIRED PARAMETERS
+
+# These values are required by the module and have no default values
+
+# --------------------------------------------------------------------
+
+# No required parameters at this time...
+
+
+# --------------------------------------------------------------------
+
+# OPTIONAL PARAMETERS
+
+# These values are optional and have default values
+
+# --------------------------------------------------------------------
+
+variable "cluster_instance_ami" {
+  type        = string
+  description = "The AMI to run on each instance in the ECS cluster."
+  default     = "ami-0e692fe1bae5ca24c"
```

```suggestion
  default     = "ami-0e692fe1bae5ca24c"
```
is this supposed to be defaulted

---

Pls ignore. I'm so sorry